CHANGELOG.md

```diff
@@ -3,6 +3,66 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.8.0
+
+_Jun 15, 2023_
+
+We'd like to offer a big thanks to the 13 contributors who made this release possible. Here are some highlights ✨:
+
+- 💫 Add a `referenceDate` prop on `DateCalendar`, `MonthCalendar` and `YearCalendar`.
```

Comment:
I would not advertise about this one until we also have the prop on the picker :+1:

Reply:
hmm, I bumped the minor because of it?

Reply:
It's a small new feature, but I don't want to bring confusion by highlighting it before all components have it

Reply:
> hmm, I bumped the minor because of it?

In that case, it sort of makes sense to release a patch if Data Grid doesn't have a change warranting a minor release. 🤔

Reply:
I think we can still do minor because of the Greek locale.